### PR TITLE
feat(documentation): integrate the google-tag-manager base script

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: packages/documentation/.storybook/manager-head.html [hunk 1]
```diff
@@ -1,7 +1,65 @@
...
+      spa: 'true', // constant, required
+      content_language: '%STORYBOOK_GTM_PAGE_CONTEXT_CONTENT_LANGUAGE%', // required
+      content_geo_region: '%STORYBOOK_GTM_PAGE_CONTEXT_CONTENT_GEO_REGION%', // required
+      source_code_version: '%STORYBOOK_GTM_PAGE_CONTEXT_SOURCE_CODE_VERSION%', // recommended
+      page_name_language_neutral: window.gtm.getPageTitle(), // required
+      page_url_language_neutral: window.location.href, // required
+      environment: window.gtm.getEnvironment(), // required
+      login_status: 'false', // required
+      platform_name: 'ms_design_system', // required
+      department_name: 'IT16.12', // required
```
Comment (imagoiq) [inline packages/documentation/.storybook/manager-head.html:43]: Maybe we can create an issue to remember to change this variable?

File: packages/documentation/package.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "bootstrap": "5.3.1"
   },
   "devDependencies": {
-    "@geometricpanda/storybook-addon-badges": "1.1.1",
+    "@geometricpanda/storybook-addon-badges": "2.0.0",
```
Comment (imagoiq) [inline packages/documentation/package.json:36]: Is it linked to this PR?

File: packages/documentation/src/stories/home.docs.mdx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import { Meta } from '@storybook/blocks';
 import { linkTo } from '@storybook/addon-links';
 import { getVersion } from '../utils/version';
+import * as HomeStories from './home.stories';
 import { Tile, Feature } from './home.blocks';
 import './home.styles.scss';
 
-<Meta title="Home" />
+<Meta of={HomeStories} />
```
Comment (imagoiq) [inline packages/documentation/src/stories/home.docs.mdx:8]: Is it linked to this PR?

**Review by imagoiq — APPROVED** [review]

File: packages/documentation/src/stories/home.docs.mdx (hunk 3 above)
Comment (oliverschuerch) [reply packages/documentation/src/stories/home.docs.mdx:8]: no

File: packages/documentation/.storybook/manager-head.html (hunk 1 above)
Comment (oliverschuerch) [reply packages/documentation/.storybook/manager-head.html:43]: It's already getting the correct value (`dev`, `int`, `prod`) by checking for the used url.

**Pull request merged**